components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.enrich.ui/src/main/resources/web/enrich-mediator/edit-mediator.jsp

```diff
@@ -89,11 +92,13 @@
     } else if (enrichMediator.getTargetType().equals("custom")) {
         targetType = CUSTOM;
         if (enrichMediator.getTargetExpression() != null) {
-            SynapseXPath targetXPath = enrichMediator.getTargetExpression();
-            NameSpacesRegistrar nameSpacesRegistrar = NameSpacesRegistrar.getInstance();
-            if (targetXPath != null) {
-                nameSpacesRegistrar.registerNameSpaces(targetXPath, "mediator.enrich.target.val_ex", session);
-                targetValue = targetXPath.toString();
+            SynapsePath synapsePath = enrichMediator.getTargetExpression();
+            if (synapsePath != null) {
```

Comment:
Don't we need to have a warning log if the synapsePath is null?

Reply:
+1 removed